### ISSUE TITLE
Some question about anchor box

### ISSUE THREAD

**Issue by SHADOWMOOON** [AlexeyAB/darknet#3230]
@AlexeyAB Hi Alexey!
When I calculate the anchor boxs(9 sets of values,the pics below).I'd like to use them in my yolo-obj.cfg  .
  
So should I replace all the anchor boxs' size in each [yolo] layer ,or should I just replace 3 sets of values in each [yolo] layer(LIke in the yolo layer where mask=6,7,8,just replace the three maximum values and in the yolo layer where mask=3,4,5,just replace the three middle value,in the mask=0,1,2,just replace 3 minimum values)?
![1558772086(1)](https://user-images.githubusercontent.com/44594124/58366663-42e0b780-7f08-11e9-93ac-92d2b3564228.png)

PS：
1. I wonder How much will the amount of pictures increase by using data augmentation in the yolo_obj.cfg(angle,saturation ,exposure ,hue)(my input numbers of pic are 598,hou many will it increase?)

2. how does the learning_rate change before and after burn_in?

Thanks in advance!

**Comment by AlexeyAB** [comment]
@SHADOWMOOON Hi,

> just replace 3 minimum values)?

You should change all 9 anchors - 9 pairs (18 values)

1. Infinity number of randomly changed images will be generated on the fly by data augmentation

2. You can read about `policy=steps` `steps=8000,9000,12000` and `scales=.1,.1,.1` https://github.com/AlexeyAB/darknet/wiki/CFG-Parameters-in-the-%5Bnet%5D-section

**Comment by SHADOWMOOON** [comment]
@AlexeyAB  Thanks for reply so soon.
since you mentioned that the number of increasing images are infinity.However,in one iteration,will all the data augmentation methods be uesd?or just Random one of them?

**Comment by AlexeyAB** [comment]
> since you mentioned that the number of increasing images are infinity.However,in one iteration,will all the data augmentation methods be uesd?or just **Random one of them** ?

Random one of them. In one iteration are used 64 images (if batch=64)

**Comment by SHADOWMOOON** [comment]
@AlexeyAB Thank you so much!
